src/__tests__/integration.test.tsx

```diff
@@ -1,16 +1,17 @@
 import { mount } from 'enzyme';
```

Comment:
Would had been nice to move to RTL, but I guess that is a bigger task

Reply:
Would be nice! It should be easier to do with these refactors though